.github/workflows/build.yml

```
@@ -0,0 +1,29 @@
+name: Build
+on: [pull_request]
+jobs:
+  botpress:
+    name: Botpress
+    runs-on: ubuntu-latest
+    steps:
+      - uses: tibdex/github-app-token@v1
+        id: generate-token
+        with:
+          app_id: ${{ secrets.BOTPRESS_BOT_APP_ID }}
+          private_key: ${{ secrets.BOTPRESS_BOT_APP_PRIVATE_KEY }}
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          token: ${{ steps.generate-token.outputs.token }}
+          submodules: true
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12.18.1'
```

```suggestion
          node-version-file: '.nvmrc'
```

Src: https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file

---

Did not know we could do that! Very nice!